.changeset/dry-buttons-exercise.md

```diff
@@ -0,0 +1,5 @@
+---
+"near-api-js": minor
+---
+
+Add support for the secp256k1 curve
```

Comment:
It would be good to add more detail here, can we please expand on this? Maybe something along the lines of: "Add support for generating secp256k1 key pairs for signing and verification of messages". Not super familiar with the changes so maybe there is more.